src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -17,279 +18,149 @@
+  We use `amp-lightbox-gallery` to implement a gallery of images.
 -->
-<div>
-    <h6>Location</h6>
-  <amp-iframe width="600" height="400"
-              layout="responsive" frameborder="0"
-              sandbox="allow-scripts allow-same-origin allow-popups"
-              src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2483.6459680494054!2d-0.14407868387219033!3d51.50136397963417!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x48760520cd5b5eb5%3A0xa26abf514d902a7!2sBuckingham+Palace!5e0!3m2!1sen!2suk!4v1469804959011"></amp-iframe>
-</div>
-<!-- ## Dynamic list -->
+<script async custom-element="amp-lightbox-gallery" src="https://cdn.ampproject.org/v0/amp-lightbox-gallery-0.1.js"></script>
 <!--
-  An `amp-list` is used to pull in the interesting attractions nearby. Learn more about the element [here](/components/amp-list).
+  We use `amp-fit-text` to fit the hotel name to the available space.
 -->
-<div>
-  <h6>Nearby attractions</h6>
-  <amp-list id="nearby-attractions"
-            width="auto"
-            height="100"
-            layout="fixed-height"
-            src="/json/nearby_attractions.json">
+<script async custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script>
+<link rel="canonical" href="<%host%>/samples_templates/hotel/">
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+<style amp-custom>
+    body {
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI ", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji ", "Segoe UI Emoji ", "Segoe UI Symbol ";
+    }
+    body > * {
+      margin: 16px;
+    }
+    .rooms [role=listitem] {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+    .rooms [role=listitem] > * {
+      margin-right: 16px
+    }
+    .rooms [role=listitem] button {
+      margin-left: auto;
+    }
+</style>
+</head>
+<body>
+  <h1 class="m1">
+    <amp-fit-text width="300" height="45" layout="responsive" max-font-size="45">
+      The Ampsterdam
+    </amp-fit-text>
+  </h1>
+  <p class="m1">The fastest hotel in town</p>
+  <h2 class="m1">When do you want to travel?</h2>
+  <!-- ## Date availability search -->
```

Comment:
suggestion: Searching for available Rooms

Reply:
done